android/capacitor/src/main/java/com/getcapacitor/CapacitorWebView.java

```diff
@@ -54,19 +56,37 @@ public boolean dispatchKeyEvent(KeyEvent event) {
         return super.dispatchKeyEvent(event);
     }
 
-    private void edgeToEdgeHandler() {
-        ViewCompat.setOnApplyWindowInsetsListener(this, (v, windowInsets) -> {
-            Insets insets = windowInsets.getInsets(WindowInsetsCompat.Type.systemBars());
+    public void edgeToEdgeHandler(Bridge bridge) {
+        String configEdgeToEdge = bridge.getConfig().adjustMarginsForEdgeToEdge();
 
-            MarginLayoutParams mlp = (MarginLayoutParams) v.getLayoutParams();
-            mlp.leftMargin = insets.left;
-            mlp.bottomMargin = insets.bottom;
-            mlp.rightMargin = insets.right;
-            mlp.topMargin = insets.top;
-            v.setLayoutParams(mlp);
+        boolean autoMargins = false;
+        boolean forceMargins = Objects.equals(configEdgeToEdge, "force");
```

Comment:
Any reason for using `Objects.equals(configEdgeToEdge,` instead of `configEdgeToEdge.equals(`?
In the rest of the codebase we use the later one.